### PR TITLE
#606 update product variantions

### PR DESCRIPTION
**Pull request by nguyenvanhadncntt**

### CODE REVIEW

**Review by chanhnguyen6975 — APPROVED** [review]

File: backoffice/modules/catalog/models/ProductVariationPost.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export type ProductVariationPost = {
+  id?: number;
```
Comment (khoahd7621) [inline backoffice/modules/catalog/models/ProductVariationPost.ts:2]: Should create type ProductVariationPut and extends ProductVariationPost like this:

export type ProductVariationPut = ProductVariationPost & { id?: number; }

**Review by khoahd7621 — CHANGES_REQUESTED** [review]
Reviewed

File: backoffice/modules/catalog/models/ProductVariationPost.ts (hunk 1 above)
Comment (nguyenvanhadncntt) [reply backoffice/modules/catalog/models/ProductVariationPost.ts]: I fixed, please reivew again

**Review by khoahd7621 — APPROVED** [review]
Reviewed

**Pull request merged**